### DISCUSSION TITLE
A way to run device commands from command line operating on groups of devices

### DISCUSSION THREAD

**Discussion by DanPartelly** [ipspace/netlab#1391] [Features]
During debugging IOL, Ive felt the need to easily run show commands on groups of devices. 
We have netlab connect -s, but it only wants to operate on one device at a time. 

Maybe worthy to add support for running on a group, with an obligatory banner containing the device name to avoid confusion.
What do you guys think ?

**Comment by ipspace** [comment]
Yeah! Go for it!

**Comment by ipspace** [comment]
Implemented
